src/components/GroupCreate/CreateButton.tsx

```diff
@@ -0,0 +1,36 @@
+import { useState } from 'react';
+import styled from 'styled-components';
+import CommonButton from '../../common/CommonButton';
+
+const CreateButton = () => {
+  const [isActive, setIsActive] = useState(false);
+
+  const handleCreateButtonClick = () => {
+    if (!isActive) {
+      setIsActive(true);
+      console.log('그룹를 생성하도록 새로운 것입니다');
+      // navigator 로 이동
+    } else {
+      console.log('에러발생');
+    }
```

Comment:
P2: 콘솔은 불필요한 코드이고, isActive가 false일 때만 true로 바뀌기 때문에, 이렇게만 구현해도 괜찮을 것 같습니다 !
```suggestion
      setIsActive(true);
      // navigator 로 이동
```